website/client/js/services/userServices.js

```diff
@@ -633,7 +633,7 @@ angular.module('habitrpg')
           var isStaticOrSocial = $window.location.pathname.match(/^\/(static|social)/);
           if (!isStaticOrSocial){
             localStorage.clear();
-            window.location.href = '/logout';
+            $window.location.href = '/logout';
```

Comment:
You may still need to stub `$window` in the test, but Karma provides a fake window for that kind of thing.